base_rest/apispec/rest_method_security_plugin.py

```diff
@@ -29,7 +29,7 @@ def operation_helper(self, path=None, operations=None, **kwargs):
         if not operations:
             return
         auth = routing.get("auth", self.spec._params.get("default_auth"))
-        if auth == "user":
+        if auth.startswith("user"):
```

Comment:
What's the motivation(s) behind this change? The strict equality is required otherwise we could match another kind of authentication that has nothing to do with the standard odoo "user" auth.

Reply:
the rationale is written in the commit msg:

> The backend might provide additional 'user' like auth policy.
> For instance, 'endpoint_route_handler' introduces 'user_endpoint'
> which raises an auth error instead of session expired error
> (which redirects to /web/login automatically).

I don't want to have a glue module just for this. Matching "user*" sounds fare as there's no non-user auth matching this naming in our ecosystem.

Reply:
Just because it doesn't exist doesn't mean we shouldn't anticipate it.

Reply:
better now?